src/PHP82.php

```diff
@@ -6,12 +6,9 @@
 
 final class PHP82 extends DefaultRules
 {
-    /**
-     * {@inheritdoc}
-     */
     public static function getRules(): array
     {
-        return array_merge(PHP81::getRules(), [
+        return array_merge(DefaultRules::getRules(), [
```

Comment:
Not sure if we should break the chain here. We currently don't have any specific version rules, but just in case... Would you still remove the versioned composition?

Reply:
To be frank, I think it was a mistake to do it the way we have done it so far.

In addition to that, the DefaultRules defined by us are enhanced by the sets [@PHP82Migration](https://mlocati.github.io/php-cs-fixer-configurator/#version:3.49).

Personally, I think it is more clear if nothing is inherited by previous versions.

Reply:
Okay, let's do it, then 🙂